deploy/charts/hostpath-provisioner/templates/storageclass.yaml

```diff
@@ -0,0 +1,11 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: {{ .Values.storageClass.name }}
+  labels:
+    {{- include "hostpath-provisioner.labels" . | nindent 4 }}
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "{{ .Values.storageClass.isDefault }}"
+provisioner: {{ .Values.storageClass.provisioner }}
+reclaimPolicy: {{ .Values.storageClass.reclaimPolicy }}
+volumeBindingMode: WaitForFirstConsumer
```

Comment:
Let's also make this configurable, and default to `WaitForFirstConsumer`. Other useful values might be `Immediate`